TMS.Net07.Homework.DaysOfWeek/TMS.Net07.Homework.DaysOfWeek/Program.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.DaysOfWeek
+{
+    class Program
+    {
+        enum daysofweek
+        {
+            Sunday,
+            Monday,
+            Tuesday,
+            Wednesday,
+            Thursday,
+            Friday,
+            Saturday
+        }
+        static void Main(string[] args)
+        {
+            daysofweek dayofweek;
+            DateTime data;
+            int day, month, year;
+            var input = "enter";
+            while (input != "exit")
+            {
+                Console.WriteLine("enter the date \"dd.mm.yyyy\" (for exit enter \"exit\")");
+                input = Console.ReadLine();
+                bool result = DateTime.TryParse(input, out data);
+                if (result == true)
+                {
+                    Console.WriteLine(data.DayOfWeek); //временнно
+                    String[] number = input.Split(new char[] { '.' }, StringSplitOptions.RemoveEmptyEntries);
+                    bool resultyear = int.TryParse(number[2], out year);
```

Comment:
For variable names we usually use `camelCase`
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841
```suggestion
                    bool resultYear = int.TryParse(number[2], out year);
```

Reply:
done